hazelcast/src/main/java/com/hazelcast/config/alto/AltoSocketConfig.java

```diff
@@ -1,58 +1,45 @@
 package com.hazelcast.config.alto;
 
-import com.hazelcast.config.InvalidConfigurationException;
-
 import javax.annotation.Nonnull;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
-public class AltoSocketConfig {
-    private static final String DEFAULT_PORT_RANGE = "11000-21000";
-    private static final int DEFAULT_RECEIVE_BUFFER_SIZE = 128 * 1024;
-    private static final int DEFAULT_SEND_BUFFER_SIZE = 128 * 1024;
+import static com.hazelcast.internal.util.Preconditions.checkPositive;
 
-    private String portRange = DEFAULT_PORT_RANGE;
-    private int receiveBufferSize = DEFAULT_RECEIVE_BUFFER_SIZE;
-    private int sendBufferSize = DEFAULT_SEND_BUFFER_SIZE;
+public class AltoSocketConfig {
```

Comment:
Similarly, missing documentation and beta tag. also, documentation for the methods and the since values

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/57f4a326887f98dfdb6329505b31d651c2376e7c